CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/StoreGraph.java

```diff
@@ -2001,11 +2002,23 @@ private void recalculateLayerVisibilities() {
     private int recalculateLayerMask(final GraphElementType elementType, final int elementId) {
         int bitmask = 0;
         if (elementType == GraphElementType.VERTEX && vertexLayerMaskAttribureId != Graph.NOT_FOUND && vertexLayerVisibilityAttributeId != Graph.NOT_FOUND) {
-            bitmask = getIntValue(vertexLayerMaskAttribureId, elementId);
+            vertexLayerMaskAttribureId = ensureAttribute(vertexLayerMaskAttribureId, 
```

Comment:
I think this was what @cygnus-x-1 mentioned the other day regarding creating a Concept for attributes used by low level classes like `StoreGraph`. If you agree, can you put a TODO just to mention that these definitions should be moved into a Concept?